hoomd/md/mesh/bond.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Mesh Bond potentials."""
+
+from hoomd.md.mesh.potential import MeshPotential
+from hoomd.data.typeparam import TypeParameter
+from hoomd.data.parameterdicts import TypeParameterDict
+
+
+class Harmonic(MeshPotential):
+    r"""Harmonic bond potential.
+
+    :py:class:`Harmonic` specifies a harmonic potential energy between all
+    particles that share an edge within the mesh. For more details see
+    :py:class:`hoomd.md.bond.Harmonic`.
+
+    Args:
+        mesh (:py:mod:`hoomd.mesh.Mesh`): Mesh data structure constraint.
```

Comment:
```suggestion
        mesh (`hoomd.mesh.Mesh`): Mesh data structure constraint.
```
This is a class anyways.